### PR TITLE
New package: ibus-bamboo-0.4.6

### PR DESCRIPTION
**Pull request by ndgnuh**
A new Vietnamese input method for ibus.

### CODE REVIEW

File: srcpkgs/ibus-bamboo/template [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Template file for 'ibus-bamboo'
+pkgname=ibus-bamboo
+version=0.4.6
+revision=1
+build_style=gnu-makefile
+short_desc="Vietnamese input method for ibus."
+maintainer="luongthanhlam"
+license="GPL-3"
```
Comment (maxice8) [inline srcpkgs/ibus-bamboo/template]: need SPDX identifier

File: srcpkgs/ibus-bamboo/template [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+homepage="https://github.com/BambooEngine/ibus-bamboo"
+distfiles="https://github.com/BambooEngine/ibus-bamboo/archive/v${version}.tar.gz>${pkgname}-${version}.tar.gz"
+checksum="2345394fd1fd8de5a9f620f3775ab34c3233c16eb6a9ffe166e3ed6f1a31f1bb"
+hostmakedepends="make go libXtst-devel libX11-devel"
```
Comment (maxice8) [inline srcpkgs/ibus-bamboo/template]: make is not required, libXtst and libX11 should be in makedepends.

hostmakedepends should go directly below build_style

File: srcpkgs/ibus-bamboo/template [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+archs=noarch
```
Comment (maxice8) [inline srcpkgs/ibus-bamboo/template]: this should go directly below revision=

File: srcpkgs/ibus-bamboo/template [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+depends="ibus"
```
Comment (maxice8) [inline srcpkgs/ibus-bamboo/template]: this should go directly below makedepends= that should go directly below hostmakedepends= that should go directly below build_style=

File: srcpkgs/ibus-bamboo/template [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+
```
Comment (maxice8) [inline srcpkgs/ibus-bamboo/template]: Remove this empty line

File: srcpkgs/ibus-bamboo/template (hunk 1 above)
Comment (ndgnuh) [reply srcpkgs/ibus-bamboo/template]: Thanks for your reply, can i get the SPDX from [here](https://spdx.org/licenses/)?

**Comment by ndgnuh** [comment]
Sorry, I should have used `xlint`. Fixed it.

File: srcpkgs/ibus-bamboo/template (hunk 1 above)
Comment (abenson) [reply srcpkgs/ibus-bamboo/template]: Yes, should be GPL-3.0-or-later

File: srcpkgs/ibus-bamboo/template [hunk 6]
```diff
@@ -0,0 +1,30 @@
+# Template file for 'ibus-bamboo'
+pkgname=ibus-bamboo
+version=0.4.6
+revision=1
+archs=noarch
```
Comment (Duncaen) [inline srcpkgs/ibus-bamboo/template]: This has to be wrong, `noarch` is for platform independent stuff, golang is a compiled language and the binaries are platform dependent.

File: srcpkgs/ibus-bamboo/template [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+hostmakedepends="git"
+makedepends="libXtst-devel libX11-devel"
+depends="ibus"
+short_desc="Vietnamese input method for ibus"
+maintainer="ndgnuh <ndgnuh99@gmail.com>"
+license="GPL-3.0-or-later"
+homepage="https://github.com/BambooEngine/ibus-bamboo"
+distfiles="https://${go_import_path}/archive/v${version}.tar.gz>${pkgname}-${version}.tar.gz"
+checksum="2345394fd1fd8de5a9f620f3775ab34c3233c16eb6a9ffe166e3ed6f1a31f1bb"
+conflicts="ibus-bamboo-git"
```
Comment (Duncaen) [inline srcpkgs/ibus-bamboo/template]: There is no `ibus-bamboo-git` package.

File: srcpkgs/ibus-bamboo/template (hunk 6 above)
Comment (ndgnuh) [reply srcpkgs/ibus-bamboo/template]: So it should be `arch=*` then?

File: srcpkgs/ibus-bamboo/template (hunk 7 above)
Comment (ndgnuh) [reply srcpkgs/ibus-bamboo/template]: I'm planning to add the git package later, but the contribution doc said i should only add one package at a time.

File: srcpkgs/ibus-bamboo/template (hunk 6 above)
Comment (Duncaen) [reply srcpkgs/ibus-bamboo/template]: The default is all architectures, so just remove it.

File: srcpkgs/ibus-bamboo/template (hunk 7 above)
Comment (ndgnuh) [reply srcpkgs/ibus-bamboo/template]: Should I add the `ibus-bamboo-git` package to this PR? I have it ready it's template file in my local repo.

Comment (Duncaen) [reply srcpkgs/ibus-bamboo/template]: We usually don't merge `-git` packages, only in very special edge cases, but I don't see this being one.

Comment (ndgnuh) [reply srcpkgs/ibus-bamboo/template]: oh, okay, i'll remove the `conflicts` line then

**Pull request merged**